yearn/apy/curve/simple.py

```diff
@@ -81,12 +83,35 @@ def simple(vault, samples: ApySamples) -> Apy:
     if vault.vault.address == "0x46AFc2dfBd1ea0c0760CAD8262A5838e803A37e5":
         boost = 1
 
-
+    # TO-DO: come up with cleaner way to deal with these new gauge rewards
     reward_apr = 0
     if hasattr(gauge, "reward_contract"):
         reward_address = gauge.reward_contract()
         if reward_address != ZERO_ADDRESS:
             reward_apr = rewards(reward_address, pool_price, base_asset_price, block=block)
+    elif hasattr(gauge, "reward_data"): # this is how new gauges, starting with MIM, show rewards
+        # get our token
+        gauge_reward_token = gauge.reward_tokens(0)
+        if gauge_reward_token == "0xEdB67Ee1B171c4eC66E6c10EC43EDBbA20FaE8e9" or gauge_reward_token == "0x0000000000000000000000000000000000000000":
```

Comment:
Maybe best to hardcode this address above similar to `YVECRV_VOTER` and `CONVEX_VOTER`